packages/x-components/src/components/scroll/scroll.mixin.ts

```diff
@@ -186,4 +218,63 @@ export default class ScrollMixin extends Vue {
       this.$emit('scroll:at-end');
     }
   }
+
+  /**
+   * Emits 'FirstItemInScrollViewChanged' event with the id of the first element that it is inside
+   * the scroll element.
+   *
+   * @internal
+   */
+  protected setFirstItemInScrollView(): void {
+    if (this.$el) {
```

Comment:
You have a problem here with `this.$el` as in the base main scroll test we're testing the scroll in the `html` and the `body`.

With this change everything works again:

```javascript
 /**
   * Emits 'FirstItemInScrollViewChanged' event with the id of the first element that it is inside
   * the scroll element.
   *
   * @internal
   */
  protected setFirstItemInScrollView(): void {
    let element: HTMLElement;
    if (this.$parent.$props['tag'] === 'html') {
      element = document.documentElement;
    } else if (this.$parent.$props['tag'] === 'body') {
      element = document.body;
    } else {
      element = this.$el as HTMLElement;
    }
    if (element) {
      const scrollTargets = element.querySelectorAll<HTMLElement>('[data-scroll-id]');

      const firstElementInView = Array.from(scrollTargets).find(element =>
        this.isAnyPartOfElementInViewport(element)
      );
      if (firstElementInView) {
        this.$x.emit('FirstItemInScrollViewChanged', firstElementInView.dataset.scrollId as string);
      }
    }
  }
```

But I'm not sure if it's the best solution, any idea?